app/shared/reducers/account/actions.js

```diff
@@ -0,0 +1,38 @@
+import * as types from '../../actions/types';
+
+const initialState = {
+  lastError: {},
+  lastTransaction: {},
+  list: [],
+  updated: null
+};
+
+export default function actions(state = initialState, action) {
+  switch (action.type) {
+    case types.RESET_ALL_STATES: {
+      return Object.assign({}, initialState);
+    }
+    case types.CLEAR_ACCOUNT_ACTIONS_CACHE: {
+      return Object.assign({}, state, {
+        list: []
+      });
+    }
+    case types.GET_ACCOUNT_ACTIONS_FAILURE: {
+      return Object.assign({}, state, {
+        lastError: action.payload.error,
+        list: []
+      });
+    }
+    case types.GET_ACCOUNT_ACTIONS_SUCCESS: {
+      return Object.assign({}, state, {
+        updated: Date.now(),
+        list: action.payload.list,
+        lastTransaction: action.payload.list[0]
```

Comment:
Why are we tracking the lastTransaction here?

Reply:
If we do proper pagination, we will need to know what was the last item that was loaded.

Reply:
It should just need to know the last index retrieved I think, as opposed to the last entire transaction.